frontend/src/app/app.module.tns.ts

```diff
@@ -4,6 +4,7 @@ import { NativeScriptModule } from 'nativescript-angular/nativescript.module';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { HomeComponent } from './home/home.component';
+import { TransactionsComponent } from './transactions/transactions.component';
```

Comment:
Don't clutter the app module. It's a good habit to make a new module for every functionality.

Reply:
Angular CLI does most of the stuff for you anyway.